index.js

```diff
@@ -0,0 +1,12 @@
+var path = require('path');
+var express = require('express');
+
+var app = express();
+var port = process.env.PORT || 3000;
+
+
+
+
+app.listen(port, function() {
+    console.log("== Listening on port", port);
+});
```

Comment:
Why the "=="?

And you might want to add a space after port so it looks nice! Like "Listening on port ".

@trilxs